runtime/arc-exceptions.js

```diff
@@ -25,3 +25,8 @@ export function removeSystemExceptionHandler(handler) {
   if (idx > -1)
     systemHandlers.splice(idx, 1);
 }
+
+registerSystemExceptionHandler((exception, methodName, particle) => {
```

Comment:
is this needed? i don't see it being used.

Reply:
@shans knows more. I believe this is used indirectly, and attaches to the outer-pec side of `proxy.raiseSystemException` (https://github.com/PolymerLabs/arcs/pull/1687/files#diff-1c7794edf33324cf4bd9b510b9e661b5R476).

Reply:
It registers an exception handler for system exceptions. Would be used only when system exceptions are generated inside the PEC